src/main/java/com/faforever/api/event/EventsService.java

```diff
@@ -12,17 +16,22 @@
 public class EventsService {
 
   private final EventRepository eventRepository;
+  private final PlayerRepository playerRepository;
```

Comment:
Services should only use repositories of entities for which they are directly responsible for. Therefore, EventsService should use PlayerService here

Reply:
done